src/mmw/js/src/modeling/views.js

```diff
@@ -66,6 +67,17 @@ var ModelingHeaderView = Marionette.LayoutView.extend({
 
     onShow: function() {
         this.reRender();
+    },
+
+    saveAfterLogin: function(user, guest) {
+        if (!guest && this.model.isNew()) {
+            var user_id = user.get('id');
+            this.model.set('user_id', user_id);
```

Comment:
How important is it to set user ID when saving models? Hopefully, we are pulling the user ID from the current session (server-side) when saving these, and not relying on POST data.